packages/chakra-ui-core/src/Progress/Progress.stories.js

```diff
@@ -1,45 +1,35 @@
 import { storiesOf } from '@storybook/vue'
-import { CSSReset, Progress as KProgress, Stack } from '..'
+import { CSSReset, Progress as KProgress, Stack, Box } from '..'
 
 storiesOf('UI | Progress', module)
-  .add('Default Progress', () => ({
-    components: { CSSReset, KProgress },
+  .addDecorator(story => ({
```

Comment:
Interesting. So we needed to add decorate before the story to fix this? Do you happen to know why? 😅

Reply:
I answered below ^^